package.json

```diff
@@ -8,7 +8,7 @@
     "lib/**/*"
   ],
   "scripts": {
-    "test": "jest --config jestconfig.json",
+    "test": "npx jest --config jestconfig.json",
```

Comment:
Maybe its better to use the jest from the bin folder `./node_modules/.bin/jest` if you think it is not common to have it globally

Reply:
This reflects a personal preference. I am inclined to avoid global installs. It just makes it a bit easier to clone and test for anyone interested. Though I use `jest` often, I didn't have it installed globally, so the tests didn't execute.

Reply:
Please resolve the conflicts you have with master